tests/Aspire.Dashboard.Components.Tests/Controls/TextVisualizerDialogTests.cs

```diff
@@ -136,16 +142,37 @@ public async Task Render_TextVisualizerDialog_ResolveTheme_LineClassesChange()
         var xml = @"<hello><!-- world --></hello>";
 
         var cut = SetUpDialog(out var dialogService);
-        await dialogService.ShowDialogAsync<TextVisualizerDialog>(new TextVisualizerDialogViewModel(xml, string.Empty), []);
+        await dialogService.ShowDialogAsync<TextVisualizerDialog>(new TextVisualizerDialogViewModel(xml, string.Empty, false), []);
+        cut.WaitForAssertion(() => Assert.True(cut.HasComponent<TextVisualizerDialog>()));
 
         Assert.NotEmpty(cut.FindAll(".theme-a11y-dark-min"));
     }
 
-    private IRenderedFragment SetUpDialog(out IDialogService dialogService, ThemeManager? themeManager = null)
+    [Fact]
+    public async Task Render_TextVisualizerDialog_WithSecret_ShowsWarningFirstOpenAsync()
```

Comment:
Add a second test that verifies the warning isn't displayed if the required value is set in local storage